### PR TITLE
docs: new playbook actions & features

### PR DESCRIPTION
**Pull request by adityathebe**
* updated
  * playbook spec
  * playbook run
  * playbook actions
  * exec action
  * templating
  * notifications
* added new checks
  * Pod
  * Git
  * HTTP
  * Notification
  * SQL
* artifacts
* webhooks

resolves: https://github.com/flanksource/docs/issues/105

### CODE REVIEW

File: mission-control/docs/playbooks/actions/exec.md [hunk 1]
```diff
@@ -1,24 +1,34 @@
 # Exec Action ![Exec Icon](https://raw.githubusercontent.com/flanksource/flanksource-ui/main/src/icons/console.svg)
```
Comment (moshloop) [inline mission-control/docs/playbooks/actions/exec.md:1]: @adityathebe Can we add examples for each action? Also use `<Icon name="console"/>`

**Comment by adityathebe** [comment]
@moshloop `static/img/icons` is placed in `.gitignore` https://github.com/flanksource/docs/blob/d775ec54c707753f652206d9999be495eb20a39a/mission-control/.gitignore#L24

Is this intentional? Those icons aren't rendered on the deploy preview.

**Comment by moshloop** [comment]
> @moshloop `static/img/icons` is placed in `.gitignore`
> 
> https://github.com/flanksource/docs/blob/d775ec54c707753f652206d9999be495eb20a39a/mission-control/.gitignore#L24
> 
> Is this intentional? Those icons aren't rendered on the deploy preview.

Yeh, just fixed Makefile to copy over the icons

File: mission-control/docs/playbooks/actions/exec.md [hunk 2]
```diff
@@ -48,12 +58,44 @@ Exec Check executes a command or script file on the target host. The type of scr
...
+| `username`    | Git auth username.                                                                                                                                             | [`types.EnvVar`](https://pkg.go.dev/github.com/flanksource/duty/types#EnvVar) |          |
+| `password`    | Git auth password.                                                                                                                                             | [`types.EnvVar`](https://pkg.go.dev/github.com/flanksource/duty/types#EnvVar) |          |
+| `certificate` | Git auth certificate.                                                                                                                                          | [`types.EnvVar`](https://pkg.go.dev/github.com/flanksource/duty/types#EnvVar) |          |
+| `destination` | Destination is the full path to where the contents of the URL should be downloaded to. If left empty, the sha256 hash of the URL will be used as the dir name. | `string`                                                                      |          |
 
-The scripts you provide will be templated using the following variables:
+## Examples
```
Comment (moshloop) [inline mission-control/docs/playbooks/actions/exec.md:82]: Can we move the example to the top and give it a filename like we do for all the canary checks

File: mission-control/docs/playbooks/concepts/templating.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 > Simply a way to represent data in different forms.
 
-Templating allows your playbook actions to work in context of a config or a component. Some actions, like exec action, can be templated using [Go templates](https://golang.org/pkg/text/template/).
+Templating allows your playbook actions to work in context of a config, health check or a component.
```
Comment (moshloop) [inline mission-control/docs/playbooks/concepts/templating.md:7]: I think lets copy  this into the actions directly, so I can see whats available from the action without needing to refer to concepts. 

For the field definitions we need to copy them into a Reference or Types page  we can refer to the go docs.

File: mission-control/docs/playbooks/concepts/run.md [hunk 4]
```diff
@@ -23,6 +23,7 @@ A run can be in one of the following states:
 | `pending`   | This is the very initial state of the run. The run is waiting for approval at this stage. |
 | `scheduled` | Once approved, the run stays in this state waiting to be executed at the schedule time.   |
 | `running`   | At this stage, the run is executing.                                                      |
+| `sleeping`  | An action on the playbook is delayed.                                                     |
```
Comment (moshloop) [inline mission-control/docs/playbooks/concepts/run.md:26]: I think this page can be merged into the playbooks.md

File: mission-control/docs/playbooks/concepts/artifacts.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# Artifacts
```
Comment (moshloop) [inline mission-control/docs/playbooks/concepts/artifacts.md:1]: This should probably have details on how to setup the artifact store and example of using it

File: mission-control/docs/playbooks/concepts/action.md [hunk 6]
```diff
@@ -4,7 +4,31 @@ Actions are the fundamental tasks executed by a playbook. A playbook can compris
 
 ## Action
 
-| Field  | Description             | Scheme       | Required |
-| ------ | ----------------------- | ------------ | -------- |
-| `name` | Name of action.         | `string`     | `true`   |
-| `exec` | Specify exec of action. | `ExecAction` | `false`  |
+| Field          | Description                                                     | Scheme                                             | Required |
```
Comment (moshloop) [inline mission-control/docs/playbooks/concepts/action.md:7]: I think merge this file  into playbook.md

**Comment by moshloop** [comment]
@adityathebe ^

**Pull request merged**